tasks/task01/narcissistic.py

```diff
@@ -0,0 +1,30 @@
+# Shagov Aleksei
+# Function to count digit in number
+def count_digit(n):
+    count = 0
+    while n:
+        n //= 10
+        count += 1
+    return count
+
+
+# Returns true if n is Narcissistic number
+def check(n):
```

Comment:
This name is not very clear 

I think it is better to use `isnarcissisticnumber` or `is_narcissistic_number`
Good example it's `isinstance` from python

You can read style in [here](https://www.python.org/dev/peps/pep-0008/)